docs/source/configuration/validation.md

```diff
@@ -0,0 +1,197 @@
+---
+myst:
+  html_meta:
+    "description": "Client side form field validation"
+    "property=og:description": "Client side form field validation"
+    "property=og:title": "Form fields validation"
+    "keywords": "Volto, Plone, frontend, React, configuration, form, fields, validation"
+---
+
+# Client side form field validation
+
+Volto provides a mechanism for delivering form field validation in an extensible way.
+This extensibility is based on the Volto component registry.
+
+## Registering a validator
+
+You can register a validator using the component registry API from your add-on configuration.
+All validators are registered under the name `fieldValidator`.
+The validators are registered using the `dependencies` array of the `registerComponent` API to differentiate the kind of validator to be registered.
+
+### `default` validators
+
+These validators are registered and applied to all fields.
+
+```ts
+config.registerComponent({
+  name: 'fieldValidator',
+  dependencies: ['default', 'minLength'],
+  component: minLengthValidator,
+});
+```
+
+It takes two `dependencies`.
+The first element should be the `default` identifier, and the second you can set it up to identify the validator.
+In the case of the example, this other dependency is `minLength`.
+It can be any string.
+
+### Per field `type` validators
+
+These validators are applied depending on the specified `type` of the field.
+
+```ts
+config.registerComponent({
+  name: 'fieldValidator',
+  dependencies: ['integer', 'maximum'],
```

Comment:
Here the `default` is missing.

Reply:
ummm, in that line, the `default` is not needed here, because it's not a default validator, it's a `type` one, so the first dependency is the name of the type.